libpd_wrapper/util/ringbuffer.c

```diff
@@ -14,6 +14,21 @@
 #include <stdlib.h>
 #include <string.h>
 
+#ifdef __APPLE__ // apple atomics
+# include <libkern/OSAtomic.h>
+# define SYNC_FETCH(ptr, val) OSAtomicOr32Barrier(val, ptr)
```

Comment:
Since val is always 0, let's just get rid of val and call OSAtomicOr32Barrier(0, ptr) directly.